app/src/main/java/com/amaze/fileutilities/home_page/database/AppDatabase.kt

```diff
@@ -124,5 +126,27 @@ abstract class AppDatabase : RoomDatabase() {
                 )
             }
         }
+
+        val MIGRATION_4_5 = object : Migration(4, 5) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                database.execSQL(
+                    "CREATE TABLE IF NOT EXISTS `AppStorageStats` " +
+                        "(`_id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, " +
+                        "`package_id` INTEGER NOT NULL, " +
+                        "`timestamp` INTEGER NOT NULL, " +
+                        "`package_size` INTEGER NOT NULL," +
+                        "FOREIGN KEY(`package_id`) REFERENCES `InstalledApps`(`_id`) ON UPDATE " +
+                        "CASCADE ON DELETE CASCADE)"
```

Comment:
Is there any reason you're keeping reference to installedApps table? 
The table is utilitized for identifying junk files. If user decides to clean all junk files, this table data is cleared, in turn clearing this storage stats as well.
More info here : https://github.com/TeamAmaze/AmazeFileUtilities/pull/157/files#diff-16dc2266dd8a0a371e6c385bf8342c4409ab5d1a11b2e26f2687dae23f541459R1975

Reply:
Suggest to keep package_name with this table itself instead of joining with second table. Unless I'm missing something this might brake.

Reply:
Ah, I thought that it just stores all installed apps. I will change my table to use the package_name then.

Reply:
Fixed it in [bd213df](https://github.com/TeamAmaze/AmazeFileUtilities/pull/141/commits/bd213dfd45133e4f38a69173623997194d09e5ef)